### ISSUE TITLE
Unable to use hyperkit driver due to PROVIDER_HYPERKIT_NOT_RUNNING

### ISSUE THREAD

**Issue by dnguyenzd** [kubernetes/minikube#13492]
### What Happened?

- Start a new cluster using hyperkit driver with following command:
```
minikube start --disk-size=40g --memory='12g' --cpus=6 --driver='hyperkit' --kubernetes-version=v1.21.4 
```
- It fails with error:
```
😄  minikube v1.25.1 on Darwin 11.6.2
    ▪ KUBECONFIG=/Users/hungphuongdavid.nguyen/.kube/config:/Users/hungphuongdavid.nguyen/.kube/kubeconfig_zendesk.yaml
🆕  Kubernetes 1.23.1 is now available. If you would like to upgrade, specify: --kubernetes-version=v1.23.1
✨  Using the hyperkit driver based on existing profile

💣  Exiting due to PROVIDER_HYPERKIT_NOT_RUNNING: /usr/local/bin/hyperkit -v failed:

💡  Suggestion: Run 'brew install hyperkit'
📘  Documentation: https://minikube.sigs.k8s.io/docs/reference/drivers/hyperkit/
```
- No problem running `/usr/local/bin/hyperkit -v`:
```
hyperkit: v0.20210107-12-gadc4ea

Homepage: https://github.com/docker/hyperkit
License: BSD
```

### Attach the log file

[log.txt](https://github.com/kubernetes/minikube/files/7938536/log.txt)

### Operating System

macOS (Default)

### Driver

Hyper-V

**Comment by klaases** [comment]
Hi @[dnguyenzd](https://github.com/dnguyenzd), were you able to get this to work?

**Comment by dnguyenzd** [comment]
@klaases no, I still haven't been able to make it work. I'm using virtualbox provider for now

**Comment by sharifelgamal** [comment]
Interesting, have you tried running `minikube delete` and `minikube start` again? It seems strange that minikube can't find hyperkit.

**Comment by klaases** [comment]
Hi @dnguyenzd – is this issue still occurring? Are additional details available? If so, please feel free to re-open the issue by commenting with `/reopen`. This issue will be closed as additional information was unavailable and some time has passed.

Additional information that may be helpful:

* Whether the issue occurs with the latest minikube release

* The exact minikube start command line used

* Attach the full output of minikube logs, run `minikube logs --file=logs.txt` to create a log file

Thank you for sharing your experience!
